ONSdigital/ras-rm-auth-service
# Fix package vulnerabilities

# What and why?

We had packages that had security issues.  This PR rebuilds the pipfile.lock to get the latest versions of things. 
This PR unpins black now that it's out of beta.  Also it reorders the makefile commands to match the other repos.

# How to test?

- Deploy code to enviornment
- Run acceptance tests
- Attempt to log in in frontstage.  Also try and change your password as a respondent.  Both of these should work just fine.

# Trello
